[2019-04-30 13:27:32,438] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.Sense_Report 2019-04-30T17:26:33.462376+00:00 [queued]>
[2019-04-30 13:27:32,441] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: Salted_Graph_Practice1.Sense_Report 2019-04-30T17:26:33.462376+00:00 [queued]>
[2019-04-30 13:27:32,441] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 13:27:32,441] {__init__.py:1354} INFO - Starting attempt 1 of 1
[2019-04-30 13:27:32,441] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-04-30 13:27:32,449] {__init__.py:1374} INFO - Executing <Task(FileSensor): Sense_Report> on 2019-04-30T17:26:33.462376+00:00
[2019-04-30 13:27:32,449] {base_task_runner.py:119} INFO - Running: ['airflow', 'run', 'Salted_Graph_Practice1', 'Sense_Report', '2019-04-30T17:26:33.462376+00:00', '--job_id', '191', '--raw', '-sd', 'DAGS_FOLDER/salted_graph.py', '--cfg_path', '/tmp/tmpihv4x6vn']
[2019-04-30 13:27:32,726] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report /home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/configuration.py:214: FutureWarning: The task_runner setting in [core] has the old default value of 'BashTaskRunner'. This value has been changed to 'StandardTaskRunner' in the running config, but please update your config before Apache Airflow 2.0.
[2019-04-30 13:27:32,726] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   FutureWarning,
[2019-04-30 13:27:32,880] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report [2019-04-30 13:27:32,880] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-04-30 13:27:33,019] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report [2019-04-30 13:27:33,019] {__init__.py:305} INFO - Filling up the DagBag from /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/dags/salted_graph.py
[2019-04-30 13:27:33,154] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report [2019-04-30 13:27:33,154] {cli.py:517} INFO - Running <TaskInstance: Salted_Graph_Practice1.Sense_Report 2019-04-30T17:26:33.462376+00:00 [running]> on host localhost.localdomain
[2019-04-30 13:27:33,162] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/reports/yelp_subset_1.csv
[2019-04-30 13:27:43,167] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/reports/yelp_subset_1.csv
[2019-04-30 13:27:53,176] {file_sensor.py:60} INFO - Poking for file /home/mjcarleb/HESWork/CSCI-E29/2019sp-airflow_project-mjcarleb/reports/yelp_subset_1.csv
[2019-04-30 13:27:53,177] {__init__.py:1580} ERROR - Snap. Time is OUT.
Traceback (most recent call last):
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
    raise AirflowSensorTimeout('Snap. Time is OUT.')
airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-04-30 13:27:53,178] {__init__.py:1611} INFO - Marking task as FAILED.
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report Traceback (most recent call last):
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/bin/airflow", line 32, in <module>
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     args.func(args)
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     return f(*args, **kwargs)
[2019-04-30 13:27:53,196] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 523, in run
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     _run(args, dag, ti)
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/bin/cli.py", line 442, in _run
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     pool=args.pool,
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/utils/db.py", line 73, in wrapper
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     return func(*args, **kwargs)
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/models/__init__.py", line 1441, in _run_raw_task
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     result = task_copy.execute(context=context)
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report   File "/home/mjcarleb/anaconda3/envs/airflow4/lib/python3.7/site-packages/airflow/sensors/base_sensor_operator.py", line 117, in execute
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report     raise AirflowSensorTimeout('Snap. Time is OUT.')
[2019-04-30 13:27:53,197] {base_task_runner.py:101} INFO - Job 191: Subtask Sense_Report airflow.exceptions.AirflowSensorTimeout: Snap. Time is OUT.
[2019-04-30 13:27:57,492] {logging_mixin.py:95} INFO - [2019-04-30 13:27:57,492] {jobs.py:2562} INFO - Task exited with return code 1
